compiler/one-cmds/how-to-use-one-commands.txt

```diff
@@ -126,7 +126,7 @@ one-import-tf
 This will convert TensorFlow model (.pb) file to our circle model. You can also
 directly call this command. one-import-tf invokes tf2tfliteV2.py script that
 will internally use TensorFlow lite converter and then invoke tflite2circle
-converter to convert tflite model to circle model. 
```

Comment:
Sorry about this change. My editor keeps remove this training space while saving this file. Can I keep this in this PR?

Reply:
If possible can you send another PR for this and below change for remove space?